delft/utilities/Embeddings.py

```diff
@@ -54,7 +56,8 @@ def __init__(self, name,
         extension='vec',
         use_ELMo=False,
         use_cache=True,
-        load=True):
+        load=True,
+        elmo_model_name=None):
```

Comment:
For my understanding, `elmo_model_name` is currently not used, right? 
Should be used from the parameter `embeddings_name`?

Reply:
Yes it is not used for the moment. By default currently the ELMo pretrained model is by language (default English, `elmo-en`). To select a particular ELMo pretrained model (like `elmo-pubmed`), the model name should be introduced by an additional distinct parameter like `elmo_model_name`.

Reply:
Understood 😄